12 confirm a booking request

tcpickard94 opened the pull request:
closes #12

RPiper93 committed:
Updated README.md

RPiper93 committed:
WIP: Wrote feature test for User Story 12

RPiper93 committed:
Merged master branch with 12-confirm-a-booking-request

RPiper93 committed:
WIP Changed the database to belong to a user instead of being many-to-many. Created a page where you can see lists of requests for your listings.

russellvaughan committed:
WIP, request confirmation failling. error undefined method session for #

tcpickard94 committed:
Working through error - removed many-to-many relationship between user and requests

tcpickard94 committed:
Completed User Story 12

Diagnosed and fixed error - the problem was due to us naming our
instance variable request which seemed to override a variable with the
same name in base.rb (this was handling our sessions and trying to call
request.session with DataMapper object)
Added a post route to update the requests status

jaxdid commented on app/models/request.rb:
```diff
@@ -4,6 +4,8 @@ class Request
+  property :status, String
+
+  belongs_to :user
   has n, :spaces, through: Resource
```
Maybe a one-to-many relationship instead of many-to-many?

tcpickard94 committed:
Updated User Story 12

Requests now belong to a space rather than have many spaces
Updated view, controllers and models accordingly

RPiper93 commented:
:shipit:
